.travis.yml

```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```

Comment:
Hi @burnumd it seems to be a problem with react-native-router-flux. It has its own react-native in its node_modules. While the line highlighted here is not needed, the line after this is needed. The @providesModule takes the reference of both the react-natives and says there is a collision. Deleting the node_module of react native router flux solves the problem